### ISSUE TITLE
Property component in state definition doesn't work

### ISSUE THREAD

**Issue by AlexSayHello** [angular-ui/ui-router#3048]
Hello, I'm new ui-router. I have a problem declaring states. When I declare a state that calls a component, if I use the syntax with the `component` property, the component is not loaded.
However, if I include the component element tag in the `template` property, this time, the component is loaded.
I Don't understand why this happens,  possibly I'm bumping with the easier thing.
I left you a  very simple example code.

``` 
<!DOCTYPE html>
<html lang="en" ng-app="app">
	<head>
		<meta charset="UTF-8">
		<title>Example</title>
	</head>
	<body>
		<ui-view></ui-view>
		<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular.min.js"></script>
        <script src="https://unpkg.com/angular-ui-router@0.3.1/release/angular-ui-router.js"></script>
        <script>
        	angular.module('app', ['ui.router'])
        		.config(function ($stateProvider) {
        			var homeState = {name: 'home', url: '/home', component: 'example'}
        			$stateProvider.state(homeState);
        		})

        		.component('example', {
        			template: '<p>Hello {{$ctrl.name}}</p>' +
        					  '<p>This is an example component</p>',
        			controller: function () {
        				this.name = 'Alex'
        			}
        		})
        </script>
	</body>
</html>
```

Thanks.

**Comment by calebeno** [comment]
The issue that I see right away is that the version of ui-router that you are using is `0.3.1`  As far as I am aware, being able to use the `component` property is only available in one of the 1.0 releases.  Try using `1.0.0-beta.3`

For future reference, I believe something like this is better suited for stack overflow.  At any rate, you might get a faster response there.

Anyway, try that and see if it works for you.

**Comment by christopherthielen** [comment]
Like @calebeno said you should be using the 1.0 beta to use route-to-component.

Switch your url to `https://unpkg.com/angular-ui-router@1.0.0-beta.3`

Here's a ui-router 1.0 plunker example: http://plnkr.co/edit/xJD0qgct6u8PaIjQ1LWc?p=info
Here's the 1.0 migration guide: https://ui-router.github.io/guide/ng1/migrate-to-1_0
